src/Components/Patient/SampleTestCard.tsx

```diff
@@ -162,8 +162,8 @@ export const SampleTestCard = (props: SampleDetailsProps) => {
             Update Sample Test Status
           </button>
           <button
-            onClick={(e) => navigate(`/sample/${itemData.id}`)}
-            className="w-full md:w-auto px-4 py-2 shadow border bg-white rounded-md border-grey-500 whitespace-nowrap text-sm font-semibold cursor-pointer hover:bg-gray-300 text-center"
+            onClick={(e) => showUpdateStatus(itemData)}
```

Comment:
@Pranshu1902 why are changing this?
@nihal467 is this an intentional change?

Reply:
I was taking about changing the navigation

Reply:
Reverted to previous version